app/locale/en.coffee

```diff
@@ -424,11 +424,13 @@
     years: "years"
 
   play_level:
+    back_to_map: "Back to Map"
     directions: "Directions"
     edit_level: "Edit Level"
     explore_codecombat: "Explore CodeCombat"
     finished_hoc: "I'm finished with my Hour of Code"
     get_certificate: "Get your certificate!"
+    ladder: "Ladder"
```

Comment:
We have `general.ladder: "Ladder"` string we can use instead of adding a new one.